__std__/nix/toolchain/library/r-overlay.nix

```diff
@@ -2,26 +2,7 @@
 
 self: super: {
 
-  rPackages = super.rPackages.override {
```

Comment:
are these overrides not needed?

Reply:
I think it has been fixed upstream with https://github.com/NixOS/nixpkgs/pull/108268#issuecomment-753688799
but I see that it was only for darwin, so maybe that is still needed for linux. Not sure how to check... ?

Reply:
Don't try and fix the R thing here until we have working CI? :) Or fix it in the main nix setup by bumping nixpkgs and check that CI is happy, then port the changes over here. If we do it now and we don't know it works it's just a landmine for us later.